web/src/components/vis/analyses.js

```diff
@@ -164,3 +164,17 @@ export default [
     icon: vuetify.icons.pca,
   },
 ];
+
+const analysisTable = (() => {
+  const table = {};
+  analysisList.forEach((entry) => {
+    table[entry.path] = entry;
+  });
+
+  return table;
+})();
```

Comment:
Could we...

1. call this `analysisMap` instead, because `table` and `list` are really the same thing to me?
2. use `Object.freeze()` before the return to prevent confusion about how this new thing should be used? 
3. use reduce instead of the IIFE?

**Note: I wrote this in github, it's untested**

```suggestion js
const analysisMap = Object.freeze(analysisList.reduce((map, entry) {
    map[entry.path] = entry;
    return map;
}, {}));
```

Reply:
Excellent recommendations. See 86f2e12 -- I did things slightly differently than you suggested.